README.md

```diff
@@ -24,3 +24,59 @@ See `const thumber` in `scthumbd.js` for now...
 ```
 npm run scthumbd
 ```
+
+
+## Envrionment variables
+
+`PORT` - listening port, defaults to 4001.
+
+`WORKERS` - number of worker processes to spawn, defaults to number of cpus detected.
+
+
+## Docker
+
+This assumes Docker Compose V2 is used - replace `docker compose` with `docker-compose` if using Docker Compose V1.
+
+Running for development:
+```
+docker compose up
+```
+will mount the local directory to the container; code changes will be reflected when the contianer is restarted.
+The container will accept requests on port 4001. To use a different port, provide a `HTTP_PORT` env, e.g.:
+
+```
+HTTP_PORT=8001 docker compose up
+```
+
+To override settings inside the container such as the number of worker or listening port, it is recommended to create another docker-compose file with the extra settings and use that file when running `docker compose`, e.g.:
```

Comment:
I don't think there's enough configuration to warrant override complexity. Especially as we're still duplicating keys in these yml files; this solution duplicates about as much code it deduplicates.

I think a greater solution would be two files: `docker-compose.dev.yml` and `docker-compose.production.yml`. Select the desired configuration using eg. `ln -s docker-compose.dev.yml docker-compose.yml` to avoid specifying the files everytime (leads to a lot of human errors).